src/views/List.jsx

```diff
@@ -18,29 +24,53 @@ export function List({ data }) {
 		setSearchTerm('');
 	};
 
+	const copyToken = () => {
+		navigator.clipboard.writeText(listToken);
+		setCopied(true);
+	};
+
 	return (
 		<>
-			<p>
-				Hello from the <code>/list</code> page!
-			</p>
-			<form onSubmit={(e) => e.preventDefault()}>
-				<label>
-					Filter Items
-					<input
-						type="text"
-						placeholder="start typing here..."
-						id="filter"
-						name="filter"
-						value={searchTerm}
-						onChange={(e) => {
-							setSearchTerm(e.target.value);
-						}}
-					/>
-				</label>
-			</form>
-			<button type="button" onClick={clearSearchTerm}>
-				clear
-			</button>
+			{data.length ? (
+				<div>
+					<label>
+						Filter Items
+						<input
+							type="text"
+							placeholder="start typing here..."
+							id="filter"
+							name="filter"
+							value={searchTerm}
+							onChange={(e) => {
+								setSearchTerm(e.target.value);
+							}}
+						/>
+					</label>
+					<button type="button" onClick={clearSearchTerm} aria-live="polite">
+						clear
+					</button>
+					<div>
+						<p>
+							Copy token to allow others join your list:
+							<button onClick={copyToken} id="token">
+								{copied ? 'Copied!' : listToken}
+							</button>
+						</p>
+					</div>
+				</div>
+			) : (
```

Comment:
I love these conditional statements to render different elements! Currently, I think the code is clear, concise, and easy to understand. However, it is getting closer to the point where I would start considering refactoring sections like the welcome message, copy token section, filter section, etc into separate components

Reply:
Definitely!

Reply:
+1 to this, Will probably be applicable in #13 if not sooner!